src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -98,7 +98,7 @@
     /**
      * The minimum range the lumberjack has to reach in order to construct or clear.
      */
-    private static final int    MIN_WORKING_RANGE      = 1;
+    private static final int    MIN_WORKING_RANGE      = 2;
```

Comment:
Why was this change made.

Reply:
The lumberjack had some distance issues, this should be two, to allow him to work with a slight difference from the trunk.